### ISSUE TITLE
FAIL is reported when case does not have 'errors' property

### ISSUE THREAD

**Issue by antonovicha** [amalfra/junit-cli-report-viewer#1]
When test case does not have errors property in XML this tool will report test case as FAIL. XML Example:

> \<testsuite name="TestCase1" timestamp="2019-02-28T08:32:30" tests="3" file="/src/test/service/test1.test.js" failures="0" time="24.577"\>
>...

Culprit is condition on this line of code:
https://github.com/amalfra/junit-cli-report-viewer/blob/master/lib.js#L40
> return **parseInt(summary.errors) === 0** && parseInt(summary.failures) === 0;

parseInt returns NaN in this case and condition will always be false.

P.S. Such report is generated by mocha junit report plugin.

**Comment by antonovicha** [comment]
Can be fixed that way:
```javascript
const isTestsuiteSuccess = (summary) => {
  const errors = parseInt(summary.errors);
  const failures = parseInt(summary.failures);
  return (isNaN(errors) || errors === 0) && (isNaN(failures) || failures === 0);
};
```

**Comment by amalfra** [comment]
@antonovicha This has been fixed in the latest release(v0.1.0). Thank you for reporting and suggesting the fix.
